cards/against-medical-advice/card.md

```diff
@@ -67,12 +67,9 @@ When patients leave AMA, providers should do whatever is possible to limit bad m
 
 ## References
 
-- Bitterman RA. Against medical advice: When should you take “no” for an answer? Lecture presented at ACEP Scientific Assembly. Chicago, Oct. 30, 2008.
+1. Bitterman RA. Against medical advice: When should you take “no” for an answer? Lecture presented at ACEP Scientific Assembly. Chicago, Oct. 30, 2008.
 
```

Comment:
Sounds found to except this card actually had in-line citations with numbers, so I change accordingly. Please advise.